tests/foreman/ui/test_contenthost.py

```diff
@@ -251,3 +260,34 @@ def test_positive_fetch_registered_by(self):
             )
             self.assertEqual(
                 result['Registered By'], self.activation_key.name)
+
+    @skip_if_bug_open('bugzilla', 1377676)
+    @skip_if_bug_open('bugzilla', 1387892)
+    @tier3
+    def test_positive_provisioning_host_link(self):
+        """Check that the host link in provisioning tab of content host page
+        point to the host details page.
+
+        @id: 28f5fb0e-007b-4ee6-876e-9693fb7f5841
```

Comment:
Tokens now changes to `:` in master. Please replace 

`@id: 4d1095c8-`

to

`:id: 4d1095c8-`

Also for other tokens

NOTE: This change is only for master, other branches still uses the `@`